grails-app/views/dashboard/_activitySummary.gsp

```diff
@@ -65,7 +65,16 @@
             ],
             "bUseRendered": false,
             "fnRowCallback": function( nRow, aData, iDisplayIndex ) {
-                $('td:eq(0)', nRow).html('<img src="${request.contextPath}/static/images/icons/silk/' + aData["type"] + '.png" />');
+                /*
+                 * We can't provide aData as a `file:` argument to resource() here,
+                 * as aData is undefined before the client browser executes this
+                 * file. The GSP parts of this file get rendered on the server
+                 * well before then, so we build the file's dirname in GSP and
+                 * its basename in JavaScript then concatenate them. :-( OBGM-65
+                 *
+                 * https://stackoverflow.com/questions/33325965/grails-gsp-accessing-variables-within-script#comment54459606_33330080
+                 */
+                $('td:eq(0)', nRow).html(`<img src="${resource(dir: '/images/icons/silk')}/` + aData['type'] + '.png" />');
```

Comment:
Sorry you had to look into that. In the future, when you hit things like this let's make sure it's code we care about keeping.

Reply:
Since there is already used string interpolation, then it could be:
```
$('td:eq(0)', nRow).html(`<img src="${resource(dir: '/images/icons/silk')}/${aData['type']}.png" />`);
```
but I think the version with adding string looks a bit cleaner.